content/developer/reference/frontend/owl_components.rst

```diff
@@ -993,3 +1011,98 @@ The shape of a `group` is the following:
       :width: 400 px
       :alt: Example of SelectMenu usage and customization
       :align: center
+
+   When SelectMenu is used with multiple selection, the `value` props must be an `Array` containing the values of the selected choices.
+
+   .. image:: owl_components/select_menu_multiSelect.png
+      :width: 350 px
+      :alt: Example of SelectMenu used with multiple selection
+      :align: center
+
+.. _frontend/tags_list:
+
+TagsList
+--------
+
+Location
+~~~~~~~~
+
+`@web/core/tags_list/tags_list`
+
+Description
+~~~~~~~~~~~
+
+This component can display a list of tags in rounded pills. Those tags can either simply list a few values, or can be editable, allowing the removal of items.
+It can be possible to limit the number of displayed items using the `itemsVisible` props. If the list is longer than this limit, the number of additional items is
+shown in a circle next to the last tag.
+
+Props
+~~~~~
+
+.. list-table::
+    :widths: 20 20 60
+    :header-rows: 1
+
+    * - Name
+      - Type
+      - Description
+    * - `displayBadge`
+      - `boolean`
+      - optional. Whether the tag is displayed as a badge.
+    * - `displayText`
+      - `boolean`
+      - optional. Whether the tag is displayed with a text or not.
+    * - `itemsVisible`
+      - `number`
+      - optional. Limit of visible tags in the list.
+    * - `tags`
+      - `array`
+      - list of `tag`'s elements given to the component.
+
+The shape of a `tag` is the following:
+
+    - `colorIndex` is an optional color id.
+    - `id` is a unique identifier for the tag.
+    - `img` is an optional image displayed in a circle, just before the displayed text.
+    - `onClick` is an optional callback that can be given to the element. This allows the parent element to handle any functionality depending on the tag clicked.
+    - `onDelete` is an optional callback that can be given to the element. This makes the removal of the item from the list of tags possible, and must be handled by the parent element.
+    - `text` is the displayed `string` associated with the tag.
+
+.. example::
+
+   In the next example, a TagsList component is used to display multiple tags.
+   It's at the developer to handle from the parent what would happen when the tag is pressed, or when the delete button is clicked.
+
+   .. code-block:: javascript
+
+      import { TagsList } from "@web/core/tags_list/tags_list";
+
+      class Parent extends Component {
+        setup() {
```

Comment:
The indentation here is not properly set: there is only 2 spaces here compared to 4 spaces below

Reply:
You're right, I copy/pasted from a test with a messed up indentation